.dockerignore

```diff
@@ -12,3 +12,5 @@ LICENSE
 README.md
 var/
 venv/
+**/__pycache__
+**/*.pyc
```

Comment:
Didn't `__pycache__` replace `.pyc` files?

Reply:
Well, in theory, all `.pyc` files should be in a `__pycache__` directory, but it was just a catch-all (you could generate the `.pyc` files manually and put them where you like).

Reply:
Hmm, can't stop people doing silly things. 😉

Reply:
(After a bit of reading, Python 3.2+ only uses stuff inside `__pycache__` anyway?)

Reply:
If we are enforcing/advising to use 3.5+ then yer its fine to remove 👍

Reply:
This base image is using 3.5.5, though this implies we will upgrade any 2.7 service to 3.5 before containerization.

Reply:
(You got me thinking, so I had to test this as it conflicted with what I thought was true)

Python 3.2+ can actually still use `.pyc` files from anywhere, it's just that Python uses `__pycache__` to manage them. Python won't generate them automatically anywhere other than a `__pycache__` (this is where they should be) but if you have valid `.pyc` elsewhere it can still be used.

This is not reason enough to warrant checking for them in this file (it's a rare edge case), it can rightfully be removed, but it is still a possibility. I could give you a whole directory of `.pyc` files that I have created and it would still run the project on your machine.

Thank you for questioning it, I have refreshed my memory of Python's cache checking order and management (a perfect start to a Monday 😁).